coffee-backend/src/main/kotlin/chat/letscoffee/meeting/MeetingController.kt

```diff
@@ -0,0 +1,62 @@
+package chat.letscoffee.meeting
+
+import chat.letscoffee.exception.ResourceNotFoundException
+import chat.letscoffee.security.security.CurrentUser
+import chat.letscoffee.security.security.UserPrincipal
+import chat.letscoffee.user.UserService
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.*
+
+
+@RestController
+@RequestMapping("/meetings")
+class MeetingController(
+    private val meetingService: MeetingService,
+    private val userService: UserService
+) {
+
+    @PostMapping("/")
+    @PreAuthorize("hasRole('ROLE')")
+    fun create(
+        @CurrentUser userPrincipal: UserPrincipal
+    ): Meeting {
+        val user = userService.getUserById(userPrincipal.id)
+        return meetingService.createMeetingRoom(user)
+    }
+
+    @DeleteMapping("/{meetingRoomId}")
```

Comment:
From the outside it looks like the object is deleted (it will no longer be available in GET endpoints). Wanted to keep the data for logging and statistics (e.g. counting how many coffee chats we've actually served). But on second thought maybe we should use another system for collecting statistics, not keeping more data than necessary... #GDPR

Reply:
Can keep it for now.